src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/CheckedListBoxItemAccessibleObjectTests.cs

```diff
@@ -0,0 +1,260 @@
+        {
+            using CheckedListBox checkedListBox = new();
+            AccessibleObject expected = checkedListBox.AccessibilityObject;
+
+            for (int index = 0; index < 3; index++)
+            {
+                checkedListBox.Items.Add(index);
+            }
+
+            AccessibleObject itemAccessibleObject = checkedListBox.AccessibilityObject.GetChild(itemIndex);
+            UiaCore.IRawElementProviderFragment actual = itemAccessibleObject.FragmentNavigate(UiaCore.NavigateDirection.Parent);
+
+            Assert.Equal(expected, actual);
+            Assert.False(checkedListBox.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void CheckedListBoxItemAccessibleObject_FragmentNavigate_NextSibling_IsExpected()
+        {
+            using CheckedListBox checkedListBox = new();
+
+            for (int index = 0; index < 3; index++)
+            {
+                checkedListBox.Items.Add(index);
+            }
+
+            AccessibleObject itemAccessibleObject1 = checkedListBox.AccessibilityObject.GetChild(0);
+            AccessibleObject itemAccessibleObject2 = checkedListBox.AccessibilityObject.GetChild(1);
+            AccessibleObject itemAccessibleObject3 = checkedListBox.AccessibilityObject.GetChild(2);
+
+            Assert.Equal(itemAccessibleObject2, itemAccessibleObject1.FragmentNavigate(UiaCore.NavigateDirection.NextSibling));
+            Assert.Equal(itemAccessibleObject3, itemAccessibleObject2.FragmentNavigate(UiaCore.NavigateDirection.NextSibling));
+            Assert.Null(itemAccessibleObject3.FragmentNavigate(UiaCore.NavigateDirection.NextSibling));
+            Assert.False(checkedListBox.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void CheckedListBoxItemAccessibleObject_FragmentNavigate_PreviousSibling_IsExpected()
+        {
+            using CheckedListBox checkedListBox = new();
+
+            for (int index = 0; index < 3; index++)
+            {
+                checkedListBox.Items.Add(index);
+            }
+
+            AccessibleObject itemAccessibleObject1 = checkedListBox.AccessibilityObject.GetChild(0);
+            AccessibleObject itemAccessibleObject2 = checkedListBox.AccessibilityObject.GetChild(1);
+            AccessibleObject itemAccessibleObject3 = checkedListBox.AccessibilityObject.GetChild(2);
+
+            Assert.Null(itemAccessibleObject1.FragmentNavigate(UiaCore.NavigateDirection.PreviousSibling));
+            Assert.Equal(itemAccessibleObject1, itemAccessibleObject2.FragmentNavigate(UiaCore.NavigateDirection.PreviousSibling));
+            Assert.Equal(itemAccessibleObject2, itemAccessibleObject3.FragmentNavigate(UiaCore.NavigateDirection.PreviousSibling));
+            Assert.False(checkedListBox.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void CheckedListBoxItemAccessibleObject_DefaultAction_IfControlIsNotCreated_ReturnsEmptyString()
```

Comment:
Please fix it in the tests below
```suggestion
        public void CheckedListBoxItemAccessibleObject_DefaultAction_IfHandleIsNotCreated_ReturnsEmptyString()
```